config/packages/easy_admin.yaml

```
@@ -0,0 +1,39 @@
+easy_admin:
+  site_name: 'NNIZER'
```

```suggestion
  site_name: '%site_name%'
```
And add this value to config/services.yaml under parameters:
site_name: 'NNIZER'